docs/modules/operation/pages/deep-dive/admin/webui/jmx-config-generator/webui.adoc

```diff
@@ -1,76 +1,80 @@
 
-== Configure JMX Connection
+== Configure JMX connection
 
-At the beginning the connection to an MBean Server of a Java application has to be configured.
+As part of the setup process, you must configure the tool's JMX connection settings.
 
-.JMX connection configuration window
-image::operation/jmx-config-generator/01-webui-connection.png[]
+.JMX connection configuration dialog
+image::operation/jmx-config-generator/01-webui-connection.png["JMX connection configuration dialog with example configuration settings"]
 
-* Service name: The name of the service to bind the JMX data collection for Collectd
-* Host: IP address or FQDN connecting to the MBean Server to load MBeans and composite data into the generation tool
-* Port: Port to connect to the MBean Server
-* Authentication: Enable/disable authentication for JMX connection with username and password
-* Skip non-number values: Skip attributes with non-number values
-* JMXMP: Enable/disable JMX Messaging Protocol instead of using JMX over RMI
+* *Service Name:* The name of the service to bind JMX data collection for collectd.
+* *Host:* The IP address or FQDN connecting to the MBean server to load MBeans and composite data into the generation tool.
+* *Port:* Port to use when connecting to the MBean server.
+* *Authentication:* Enable or disable authentication for the JMX connection.
+If enabled, the connection will need to be authenticated with a username and password.
+* *Skip Non-Number Values:* Skip attributes with non-number values.
+* *JMXMP:* Enable or disable the JMX Messaging Protocol.
+If enabled, the tool will use the JMX Messaging Protocol instead of using JMX over RMI.
 
-By clicking the arrow ( > ) the MBeans and composite data will be retrieved with the given connection settings.
-The data is loaded into the MBeans Configuration screen that lets you select metrics for the data collection configuration.
+After you enter your configuration settings, click the *arrow* symbol (*>*) to retrieve MBeans and composite data.
+The data is loaded into the MBeans Configuration.
+From there, you can select metrics to include in data collection.
 
-== Select MBeans and Composite
+== Select MBeans and composite data
 
-The MBeans Configuration section is used to assign the MBean and composite data attributes to RRD domain specific data types and data source names.
+The MBeans Configuration dialog lets you assign MBeans and composite data attributes to RRD domain-specific data types and data source names.
 
-.Select MBeans or Composite Data for {page-component-title} data collection
-image::operation/jmx-config-generator/02-webui-mbean-selection.png[]
+.Select MBeans and composite data attributes for {page-component-title} data collection
+image::operation/jmx-config-generator/02-webui-mbean-selection.png["MBeans Configuration dialog displaying a list of MBeans and composite data attributes. Some are selected and have been assigned aliases"]
```

Comment:
```suggestion
image::operation/jmx-config-generator/02-webui-mbean-selection.png["MBeans Configuration page displaying a list of MBeans and composite data attributes. Some are selected and have been assigned aliases."]
```
Not sure if we've been consistent, but I noticed that with two sentences, there tended to be a period at the end, with one sentence, no period.